docs/user-guide/how_to_use_volcano_vgpu.md

```diff
@@ -116,6 +116,76 @@ You can validate device memory using nvidia-smi inside container:
 > the GPUs on the machine will be exposed inside your container.
 > The number of vgpu used by a container can not exceed the number of gpus on that node.*
 
+### Use NVIDIA dynamic MIG
+
+The detailed design is at [dyanamic mig](https://github.com/volcano-sh/volcano/blob/master/docs/design/dynamic-mig.md).
+
+#### Configure the volcano scheduler
+
+After installing the [volcano vgpu device plugin](https://github.com/Project-HAMi/volcano-vgpu-device-plugin), a configmap about NVIDIA MIG geometry definition will be created. The volcano scheduler relies on this configuration for scheduling decisions.
+
+Add deviceshare.KnowGeometriesCMName and deviceshare.KnowGeometriesCMNamespace to the scheduler configmap.
+
+```shell script
+kubectl edit cm -n volcano-system volcano-scheduler-configmap
+```
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: volcano-scheduler-configmap
+  namespace: volcano-system
+data:
+  volcano-scheduler.conf: |
+    actions: "enqueue, allocate, backfill"
+    tiers:
+    - plugins:
+      - name: priority
+      - name: gang
+      - name: conformance
+    - plugins:
+      - name: drf
+      - name: deviceshare
+        arguments:
+          deviceshare.VGPUEnable: true # enable vgpu
+          deviceshare.KnowGeometriesCMName: volcano-vgpu-device-config
```

Comment:
KnownGeometriesCMName?

Reply:
deviceshare.KnowGeometriesCMName is aligned with other plugin configurations.

Reply:
Done